web/static/css/styles.css

```diff
@@ -0,0 +1,210 @@
+    U+2020, U+20A0-20AB, U+20AD-20C0, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+@font-face {
+  font-family: "Public Sans";
+  font-style: italic;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/publicsans/v18/ijwTs572Xtc6ZYQws9YVwnNDTJzax8s3Jik.woff2)
+    format("woff2");
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+    U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191,
+    U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+@font-face {
+  font-family: "Public Sans";
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/publicsans/v18/ijwRs572Xtc6ZYQws9YVwnNIfJ7QwOk1Fig.woff2)
+    format("woff2");
+  unicode-range: U+0100-02AF, U+0304, U+0308, U+0329, U+1E00-1E9F, U+1EF2-1EFF,
+    U+2020, U+20A0-20AB, U+20AD-20C0, U+2113, U+2C60-2C7F, U+A720-A7FF;
+}
+@font-face {
+  font-family: "Public Sans";
+  font-style: normal;
+  font-weight: 100 900;
+  font-display: swap;
+  src: url(https://fonts.gstatic.com/s/publicsans/v18/ijwRs572Xtc6ZYQws9YVwnNGfJ7QwOk1.woff2)
+    format("woff2");
+  unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
+    U+02DC, U+0304, U+0308, U+0329, U+2000-206F, U+2074, U+20AC, U+2122, U+2191,
+    U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
+}
+header {
+  > div.utility-header {
+    background-color: #f5f9fa;
+    > div.container > div.flex-row {
+      > div.social-media-links
+        > div.official-collapsible-content
+        > button.btn-official {
+        border: none;
+        font-size: 0.875rem;
+        background: transparent;
+        text-align: left;
+        text-wrap: nowrap;
+        &[aria-expanded="true"] {
+          > span.ca-gov-icon-caret-up {
+            display: inline-block;
+          }
+          > span.ca-gov-icon-caret-down {
+            display: none;
+          }
+        }
+        &[aria-expanded="false"] {
+          > span.ca-gov-icon-caret-up {
+            display: none;
+          }
+          > span.ca-gov-icon-caret-down {
+            display: inline-block;
+          }
+        }
+      }
+      @media (width <= 770px) {
+        flex-direction: column !important;
+        > div.settings-links {
+          margin-left: 0 !important;
+          justify-content: flex-start;
+          > a {
+            margin-left: 0;
+            margin-right: 1rem;
+            font-size: 0.8rem;
+          }
+        }
+      }
+      @media (width <= 350px) {
+        > div.social-media-links {
+          flex-direction: column;
+        }
+      }
+    }
+  }
+  > div.section-default > div.branding > div.header-organization-banner {
+    display: flex;
+    margin-right: 0;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    > a > div.logo-assets {
+      > img.logo-img {
+        max-height: 60px;
+        height: 60px;
+        aspect-ratio: 18/5;
+      }
+      > div.logo-text {
```

Comment:
Appears to be unused

Reply:
this is cleaned. Ntotext is needed here. @thekaveman